src/CodeGen_Hexagon.cpp

```diff
@@ -577,19 +577,23 @@ void CodeGen_Hexagon::push_buffer(const std::string &name, int dimensions, llvm:
         // extern stages on hexagon that take buffers passed through from
```

Comment:
could you point me at information/examples on this feature ?
just curious... i think we might have internal users interested in something like this.